### ISSUE TITLE
[Bug] Error: Cannot find module './lib/extend'

### ISSUE THREAD

**Issue by phroggyy** [jessepollak/card#197]
When attempting to install Card, I receive plenty of errors:

```
[Error: Cannot find module './lib/extend' from '/Users/user/directory/project/node_modules/card/lib/js']
``` 
```
[Error: Cannot find module '../scss/card.scss' from '/Users/user/directory/project/node_modules/card/lib/js']
```
```
[Error: Cannot find module './payment/src/payment.coffee' from '/Users/user/directory/project/node_modules/card/lib/js']
```
```
[Error: Cannot find module 'is' from '/Users/user/directory/project/node_modules/card/lib/js']
```
```
[Error: Cannot find module 'sassify' from '/Users/user/directory/project/node_modules/card/lib/js']
```

Is there anyone else that has encountered these issues? I'm lacking a lot of experience with NPM, so I'm thinking this might be some very obvious issue I've missed?

**Comment by manioube** [comment]
in the directory where you installed Card, did you then run 'npm install' to install all the dependencies?
what method did you use to install Card?

**Comment by phroggyy** [comment]
@manioube yep, I installed it by `npm install card --save`

**Comment by ebuster** [comment]
Have the same issue. Unable to use this lib my Webpack bundle.

**Comment by duluca** [comment]
I don't see any errors during installation, but when requiring the package through browserify, I get `Error: Cannot find module './lib/extend'`.

**Comment by jessepollak** [comment]
Fix will go out in `v1.2.2`.
